parts/linux/cloud-init/artifacts/nbcparser.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -x
+# TODO: propagate exit status to bootstrapper
+if [ ! -f /opt/azure/containers/nbc.json ]; then 
+    exit 1
+fi
+/opt/azure/containers/nbcparser /opt/azure/containers/nbc.json > cse_cmd.sh
```

Comment:
`/opt/azure/containers/nbcparser` will produce logs to stdout/err right? should we pipe that output to a dedicated log file? maybe something like `/var/log/azure/aks/bootstrap.log`

Reply:
added!